libraries/ms-drivers/src/ltc_afe_impl.c

```diff
@@ -217,7 +252,8 @@ StatusCode ltc_afe_impl_trigger_aux_conv(LtcAfeStorage *afe, uint8_t device_cell
   // we use GPIO2, GPIO3, GPIO4, GPIO5 to select which input to read
```

Comment:
Is this now resolved? `// TODO(SOFT-9): Update GPIO usage to match updated design`